test/app-tap/suite.ini

```diff
@@ -4,7 +4,7 @@ description = application server tests (TAP)
 lua_libs = lua/require_mod.lua lua/serializer_test.lua lua/process_timeout.lua
 is_parallel = True
 use_unix_sockets_iproto = True
-release_disabled = gh-5040-inter-mode-isatty-via-errinj.test.lua
+release_disabled = gh-5040-inter-mode-isatty-via-errinj.test.lua gh-2717-no-quit-sigint.test.lua
```

Comment:
please add an explanation why test should be skipped on release build

Reply:
Added.